### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by FYNIXqwq** [Cryolite/kanachan#29]
As the readme of this repository indicates, plentiful game log is necessary in the preparation of training. Why this model does not randomly shuffle and generate the deck (ピーパイとワンパイJapanese), then generate 4 agents as players?

**Comment by Cryolite** [comment]
Are you asking, "Why doesn't this repository use training based on the game records obtained from self-play?"

**Comment by FYNIXqwq** [comment]
Yes. I am confusing that why multiple agents self-play with reward is not enough. Card game models of imperfect information game i.e. DouZero is a complete self-play model, and it works well.
